src/components/DownloadContactListButton.tsx

```diff
@@ -15,14 +15,13 @@ export function DownloadContactListButton (props: DownloadButtonProps) {
     const filteredPropObject = getUniqueListBy(arrayOfPropertyObjects, 'id')
     filteredPropObject.forEach(result => {
         if (typeof result.owner_details !== 'undefined' && Array.isArray(result.owner_details)) {
-            const mobile = result.owner_details[0].Mobile
-            const workPhone = result.owner_details[0].Work_Phone
+            const mobile = result.owner_details[0].Mobile || result.owner_details[1]?.Mobile
```

Comment:
can there be more than two owners? Might need to change to use owner_details.map((owner) => owner.Mobile)